packages/react-components/src/index.ts

```diff
@@ -55,7 +73,42 @@ export type {
   StrokeWidthTokens,
   Theme,
 } from '@fluentui/react-theme';
-export { SSRProvider } from '@fluentui/react-utilities';
+export {
+  clamp,
+  getNativeElementProps,
+  getNativeProps,
+  getPartitionedNativeProps,
+  getRTLSafeKey,
+  getSlots,
+  omit,
+  resolveShorthand,
+  shouldPreventDefaultOnKeyDown,
+  useBoolean,
+  useConst,
+  useControllableState,
+  useId,
```

Comment:
```suggestion
  useId,
  resetIdsForTests,
```

On the other hand we genuinely want to export this since partners might want deterministic ids in tests

Reply:
Added back :)